src/02-common-principles.md

```diff
@@ -644,6 +540,150 @@ for more information.
 }
 ```
 
+## The Inheritance Principle
+
+1.  Any metadata file (such as `.json`, `.bvec` or `.tsv`) MAY be defined at any directory level.
+
+1.  For a given data file, any metadata file is applicable to that data file if:
+    1.  It is stored at the same directory level or higher;
+    1.  The metadata and the data filenames possess the same suffix;
+    1.  The metadata filename does not include any entity absent from the data filename.
+
+1.  A metadata file MUST NOT have a filename that would be otherwise applicable
+    to some data file based on rules 2.2 and 2.3 but is made inapplicable based on its
+    location in the directory structure as per rule 2.1.
+
+1.  There MUST NOT be multiple metadata files applicable to a data file at one level
+    of the directory hierarchy.
+
+1.  If multiple metadata files are applicable to a given data file:
+
+    1.  For [tabular files](#tabular-files) and other simple metadata files
+        (for instance, [`bvec` / `bval` files for diffusion MRI](#04-modality-specific-files/01-magnetic-resonance-imaging#required-gradient-orientation-information),
+        only the file lowest in the filesystem hierarchy SHALL be treated as being
```

Comment:
could we use MUST instead of SHALL here? Simply because so far there is zero mention of SHALL in the spec, but lots of MUST - and I thought these were equivalent?

Reply:
I had thought intuitively that there was a distinction between MUST and SHALL, but that appears to be incorrect based on RFC2119. My thinking here was that it's entirely plausible that an app may have no interest in the content of a sidecar file, and just load an e.g. image in isolation, and therefore stating that such sidecar information MUST be loaded is to me unnecessarily dictatorial. It's more like: if you are loading a data file, *and* you are interested in the sidecar data associated with that file, *then* only the lowest sidecar file MUST be utilised.

Reply:
See if you're happy with 8e808a12.

Reply:
yes, thank you - resolving.